src/app/components/MyLibrary.tsx

```diff
@@ -0,0 +1,109 @@
+'use client';
+
+import { BsFileMusic, BsFileMusicFill } from 'react-icons/bs';
+import { CiCirclePlus } from 'react-icons/ci';
+
+import { useQuery } from '@tanstack/react-query';
+import Image from 'next/image';
+
+interface Props {
+  width: number;
+  setWidth: React.Dispatch<React.SetStateAction<number>>;
+}
+
+type Library_data = {
```

Comment:
typescript에서 type과 interface의 네이밍은 class 네이밍과 마찬가지로 일반적으로 PascalCase로 네이밍 합니다.
```suggestion
type LibraryData = {
```